nextjs/text-to-speech/next.config.ts

```diff
@@ -0,0 +1,7 @@
+import type { NextConfig } from "next";
+
+const nextConfig: NextConfig = {
+  /* config options here */
```

Comment:
```suggestion
  export: 'standalone',
```

makes build faster, smaller